homeassistant/components/fritzbox/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
```

Comment:
I see in climate.py we have some extra state attributes, would it make sense to create separate entities out of it?

Reply:
Extra bonus points in climate: make the min and max temp shorthand